src/closures.c

```diff
@@ -34,6 +34,82 @@
 #include <ffi.h>
 #include <ffi_common.h>
 
+#ifdef __NetBSD__
+#include <sys/param.h>
+#endif
+
+#if __NetBSD_Version__ - 0 >= 799007200
```

Comment:
I would put <sys/mman.h> next to <sys/param.h> include and later check for:
`#if defined(__NetBSD__) && defined(PROT_MPROTECT)`

Reply:
Seems like a reasonable approach.